pdns/dnsdistdist/dnsdist-configuration-yaml.cc

```diff
@@ -143,6 +143,18 @@ static uint8_t strToRCode(const std::string& context, const std::string& paramet
   return *rcode;
 }
 
+static uint16_t strToQType(const std::string& context, const std::string& parameterName, const ::rust::String& qtype_rust_string)
+{
+  auto qtype_str = std::string(qtype_rust_string);
+  boost::to_lower(qtype_str);
+  QType qtype;
+  qtype = std::string(qtype_str);
```

Comment:
Not sure an explicit cast is needed here.
```suggestion
  qtype = qtype_str;
```